.eslintrc.cjs

```diff
@@ -43,7 +43,7 @@ module.exports = {
 
     // Check for mandatory file extensions
     // https://nodejs.org/api/esm.html#mandatory-file-extensions
-    'import/extensions': ['error', 'ignorePackages'],
+    'import/extensions': ['error', 'always', { ignorePackages: true }],
```

Comment:
@feedmypixel Can I quickly check, would this now prevent package `*.js` imports?

When extending Nunjucks, it's quite common to import the filters directly:

```js
import filters from 'nunjucks/src/filters.js'

filters.capitalize()
filters.indent()
filters.safe()
```

If not, don't worry, just seeing if we need this change too (not tried it yet!)

Reply:
No thats fine - see final clause here - https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md#examples

Reply:
Brill, thanks for confirming

Wonder if I left off `'always'` because of this bit at the bottom?

>In the future, when this rule supports native node ESM resolution, and the plugin is configured to use native rather than transpiled ESM (a config option that is not yet available) - setting this to `always` will have no effect.

Reply:
oh thats interesting also